digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleLinearAssetDao.scala

```diff
@@ -1006,4 +1006,17 @@ class OracleLinearAssetDao(val vvhClient: VVHClient, val roadLinkService: RoadLi
          and ca.created_date > ADD_MONTHS(TO_DATE(TO_CHAR(${lastCreationDate.get}, 'YYYY-MM-DD'), 'YYYY-MM-DD hh24:mi:ss'), -1)
          #$municipalityFilter""".as[(Long, Int)].list
   }
+
+  def getLinksWithExpiredAssets(linkIds: Seq[Long], assetType: Int): Seq[Long] = {
+    val withLinkIds = if (linkIds.nonEmpty) s"and pos.link_id in (${linkIds.mkString(",")})" else ""
```

Comment:
shouldn't this IN be replaced with MassQuery ?

Reply:
In this case, we use the IN clause because it's supposed to return even if the link ids Sequence was empty, and if we do that using the mass query we never get results.